src/data/data-gig-detail.html

```diff
@@ -24,17 +30,40 @@
         observer: '_rawChanged'
       },
 
+      gigs: {
+        type: Object,
+        observer: '_gigsChanged'
+      },
+
       gig: {
         type: Object,
         notify: true
       }
     },
 
+    ready: function() {
+      console.log(this.$.document)
+    },
+
     _rawChanged: function(newValue) {
       // Only set `gig` if there's actually a value there.
       if (!this.$.document.valueIsEmpty(newValue)) {
         this.set('gig', newValue);
       }
+    },
+
+    _gigsChanged: function(gigs) {
+      if (!this.$.document.online) {
+        var gigId = this.gigId;
+        var gig = gigs.filter(function(gig) {
```

Comment:
I think I originally had this just reuse the gig list data because in theory we already had it locally so searching for it locally was probably faster than making a network request for it specifically.

Still, here I'd prefer to find it in a way that can stop as soon as it is found instead of continuing to iterate through the rest of the array to possibly find more that are explicitly thrown away later when you grab the `0` index anyway.